webapi/src/main/scala/org/knora/webapi/Main.scala

```diff
@@ -29,15 +29,15 @@ object Main extends App with LiveCore with KnoraService  {
     //Kamon.start()
 
     /* Check and wait until all actors are running */
-    checkActorSystem
+    checkActorSystem()
```

Comment:
Did you do that because the method has side-effects? http://docs.scala-lang.org/style/method-invocation.html